docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -41,16 +41,16 @@
 |spring.cloud.gateway.filter.rewrite-path.enabled | `true` | Enables the rewrite-path filter.
 |spring.cloud.gateway.filter.rewrite-response-header.enabled | `true` | Enables the rewrite-response-header filter.
 |spring.cloud.gateway.filter.save-session.enabled | `true` | Enables the save-session filter.
-|spring.cloud.gateway.filter.secure-headers.content-security-policy | `default-src 'self' https:; font-src 'self' https: data:; img-src 'self' https: data:; object-src 'none'; script-src https:; style-src 'self' https: 'unsafe-inline'` |
-|spring.cloud.gateway.filter.secure-headers.content-type-options | `nosniff` |
+|spring.cloud.gateway.filter.secure-headers.content-security-policy | `default-src 'self' https:; font-src 'self' https: data:; img-src 'self' https: data:; object-src 'none'; script-src https:; style-src 'self' https: 'unsafe-inline'` | 
+|spring.cloud.gateway.filter.secure-headers.content-type-options | `nosniff` | 
```

Comment:
I wasn't sure what to do with these `clean install -P docs -pl docs -DskipTests` generates it, I guess this is the right format.